construct-sturdy-uis-with-xstate/notes/07-save-previous-state-with-history-state.md

```diff
@@ -0,0 +1,26 @@
+# Save Previous States with a History State
+
+- [Video Link](https://egghead.io/lessons/react-save-previous-states-with-a-history-state)
+- [Code Link](https://github.com/isaacplmann/sturdy-uis/tree/lesson6-end)
+
+
+🔑 A _[history](https://xstate.js.org/docs/guides/history.html#history) state_ node is a special kind of state node that, when reached, tells the machine to go to the last state value of that region. There's two types of history states:
+- 'shallow', which specifies only the top-level history value.
+- 'deep', which specifies the top-level and all child-level history values.
+
+
+📹 History states explanation:
+<iframe width="560" height="315" src="https://www.youtube.com/embed/uczggW1PF6I" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Most of the notes are consumed directly on GitHub - `iframes` are not accessible in Markdown. My suggestion is to add a link instead.

Reply:
Will do, adding right now